views/confirmation.html

```diff
@@ -36,16 +36,15 @@ <h2 class="govuk-heading-m">What you need to do now</h2>
       {% endif %}
 
       <h2 class="govuk-heading-m">What happens next</h2>
-      <p class="govuk-body">We'll email you to let you know if your application to register an overseas entity has been accepted or rejected. We'll process the application as soon as possible. However, because this is a new service, we're not yet able to tell you exactly how long this will take.</p>
+      <p class="govuk-body">We'll email you to let you know if your application to {{ "update" if isUpdate else "register" }} an overseas entity has been accepted or rejected. We'll process the application as soon as possible. However, because this is a new service, we're not yet able to tell you exactly how long this will take.</p>
```

Comment:
These three new bits of logic in the view could be verified/tested in the registration and update controller tests by checking that the correct text is output. Probably just a few new assertions to add to an existing test ("renders the confirmation page for agent") in the case of 'registration'.

Reply:
good idea - added